datasets.py

```diff
@@ -69,7 +68,9 @@ def __getitem__(self, idx):
         label = self.items[idx][1]
         img_path = os.path.join(self.dir, img_name)
         img = Image.open(img_path)
-        img = self.transform(img)
+        if self.transform:
+            img = self.transform(img)
+        img = transforms.ToTensor()(img)
```

Comment:
It's better to use ToTensor() transform as a part of self.transform(img) call. Just construct more complex transform with transform.Compose()